climakitae/data_loaders.py

```diff
@@ -58,18 +60,23 @@ def _open_and_concat(file_list, selections, ds_region):
             )
         )
         # subset data spatially:
+        data_crs = ccrs.LambertConformal(central_longitude=-70,central_latitude=38,
+                                 standard_parallels=(30.0,60.0))
+
         if ds_region:
+            output = data_crs.transform_points(ccrs.PlateCarree(),
+                                                   x=ds_region.coords[0][:,0],
+                                                   y=ds_region.coords[0][:,1])
+
+            data = data.sel(x=slice(np.nanmin(output[:,0]), np.nanmax(output[:,0])),
```

Comment:
how does nanmin and nanmax work here? does it only select coordinates without nans?

Reply:
It ignores the nans, whereas the default min, max, etc does not. Numpy as a nan-ignoring version of all the standard basic stats.